client/state/jetpack-connect/test/reducer.js

```diff
@@ -313,6 +313,30 @@ describe( 'reducer', () => {
 			expect( state[ 'website.com' ] ).to.have.property( 'timestamp' )
 				.to.be.at.least( nowTime );
 		} );
+
+		it( 'should persist state', () => {
+			const originalState = deepFreeze( {
+				ssoUrl: 'https://website.com?action=jetpack-sso&result=success&sso_nonce={$nonce}&user_id={$user_id}',
+				siteUrl: 'https://website.com'
```

Comment:
I'll address changing `website.com` in a separate PR, as it occurs on several other locations as well.